lib/auth_41.js

```diff
@@ -57,17 +57,15 @@ function xor(a, b) {
   return result;
 }
 
+exports.xor = xor;
+
 function token(password, scramble1, scramble2) {
   // TODO: use buffers (not sure why strings here)
   if (!password) {
     return Buffer.alloc(0);
   }
   const stage1 = sha1(password);
-  return exports.calculateTokenFromPasswordSha(
-    stage1,
-    scramble1,
-    scramble2
-  );
+  return exports.calculateTokenFromPasswordSha(stage1, scramble1, scramble2);
```

Comment:
I know I did not touch these, so I suppose there must be pre commit hooks to do some auto formatting.

Reply:
yes, there is prettier percommit hook

Reply:
via lint-staged